Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -11,6 +11,7 @@ import UIKit
 @available(iOS 13.0, *)
 internal protocol DebugViewDelegate: AnyObject {
     func debugView(did event: DebugView.Event)
+    func screenCaptured()
```

Comment:
This is fine as is, but another option would be to add `screenCapture` as a case to `DebugView.Event`.

Reply:
Good call, making consistent with others.